src/nuklear_font.c

```diff
@@ -2152,9 +137,9 @@ nk_font_baker_memory(nk_size *temp, int *glyph_count,
             *glyph_count += nk_range_glyph_count(i->range, range_count);
         } while ((i = i->n) != iter);
     }
-    *temp = (nk_size)*glyph_count * sizeof(struct nk_rp_rect);
-    *temp += (nk_size)total_range_count * sizeof(struct nk_tt_pack_range);
-    *temp += (nk_size)*glyph_count * sizeof(struct nk_tt_packedchar);
+    *temp = (nk_size)*glyph_count * sizeof(struct stbrp_rect);
+    *temp += (nk_size)*glyph_count * sizeof(stbtt_pack_range);
```

Comment:
Overall I think we're getting close to merging. But this change from `` to `` is overly suspicious to merge it :cry:.

I think stb_truetype started using some structs a bit differently in the newer versions. Maybe it's some off-by-one oversight or struct padding or whatever. I'd look at how the structs `nk_rp_rect` `nk_tt_pack_range` `nk_tt_packedchar` `stbrp_rect` `stbtt_pack_range` `stbtt_packedchar` `nk_font_bake_data` are being used everywhere and whether they're used everywhere with corresponding counts as I don't think `stbtt_pack_range` depends on glyph count, but rather truly on the calculated `total_range_count`.

Have you tried just adding `time += 1` or `time += 8` or similar (i.e. a very small fixed number)?